py/loqui/encoders.py

```diff
@@ -27,7 +27,7 @@ class _erlpack:
         dumps = erlpack.pack
         loads = erlpack.unpack
 
-    ENCODERS['erlpack'] = _erlpack
+    ENCODERS[b'erlpack'] = _erlpack
```

Comment:
requiring these keys to be bytestrings without enforcing it anywhere seems kinda dangerous. Any ideas?